docs/website/docs/intro.md

```diff
@@ -43,15 +43,22 @@ load_info = pipeline.run(data, table_name='player')
 ```
 <!--@@@DLT_SNIPPET_END index-->
 
-Now copy this snippet to a file or a Notebook cell and run it. If you do not have it yet, install **duckdb** dependency (default `dlt` installation is really minimal):
+If you don't have `duckdb`, you can install it as an extra:
 ```sh
 pip install "dlt[duckdb]"
 ```
+Now **run** your Python file or Notebook cell.
 
-How the script works?: It extracts data from a
-[source](general-usage/glossary.md#source) (here: **chess.com REST API**), inspects its structure to create a
-[schema](general-usage/glossary.md#schema), structures, normalizes and verifies the data, and then
-loads it into a [destination](general-usage/glossary.md#destination) (here: **duckdb** into a database schema **player_data** and table name **player**).
+What that code does:
+
+1. Extracts data from a
+[source](general-usage/glossary.md#source) (here: **chess.com REST API**)
```

Comment:
could you add the dots "." at the end of sentences
1. Extracts data from a
[source.]
2. `dlt` inspects the data's structure to create a
[schema.]

Reply:
Done